tools/check-email.sh

```diff
@@ -18,8 +18,8 @@ handle_file
 for email_domain in `cat $filename`
   do
     domain=`echo $email_domain |cut -d"@" -f2`
-    check_result=`dig mx $domain |grep MX |wc -l`
-    if (( $check_result > 1 )); then
+    check_result=`dig MX $domain | grep "^[^;].*MX | wc -l`
```

Comment:
Missing double quote in first grep:
check_result=`dig MX $domain | grep "^[^;].*MX" | wc -l`